test/manual/amp-ad.yieldbot.amp.html

```diff
@@ -27,7 +27,7 @@
           data-psn="1234"
           data-yb-slot="medrec"
           data-slot="/2476204/medium-rectangle"
-          data-multi-size="300x220,300x200"
+          data-multi-size="300x220,300x210, 300x200,300NNN,300xNNN,"
```

Comment:
Yes. The invalid multi-size attribute is for testing only.

The genesis of _test/manual/amp-ad.yieldbot.amp.html_ is from here:

- https://github.com/ampproject/amphtml/pull/2850#r59041799

Since `utils.getMultiSizeDimensions()` is now used, there are some unit tests for the function, albeit few dirty attribute data tests. However, risk is marginal I'd say.

Therefore, we could jettison the Yieldbot manual test page.

However, should an ad network wish to include unit tests for their implementation is there a preferred approach for `amp-ad` types that are not a4a extensions? I would want to avoid polluting the example with testing cruft.